react/features/calendar-sync/web/CalDav.ts

```diff
@@ -0,0 +1,72 @@
+import { IStore } from '../../app/types';
+import {
+  getCalendarEntries,
+  loadCalDavAPI,
+  signIn,
+  updateCalendarEvent,
+  updateProfile
+} from '../../caldav-api/actions'; // @ts-ignore
+import calDavApi from '../../caldav-api/calDavApi.web';
```

Comment:
Did you forget to include this in the PR?

Reply:
Respected Sir,
I am having trouble in completing "from '../../caldav-api/actions'; // @ts-ignore
import calDavApi from '../../caldav-api/calDavApi.web';"
Kindly help me through this.

Reply:
The reason that this was not implemented long time ago is missing a good js library  that implements caldav. Implementing and supporting a library like that is a lot of work and should happen outside of jitsi-meet.

Reply:
So is there any other possible way to do it?
I want to work on it and complete it.

Reply:
Start simple. Find a library or implement something that is being able to communicate  caldav outside jitsi-meet. A simple example. When you have it then try to integrate it in jitsi-meet.